ProcessLib/ProcessVariable.h

```diff
@@ -57,6 +57,9 @@ class ProcessVariable
 	/// Returns a mesh on which the process variable is defined.
 	MeshLib::Mesh const& getMesh() const;
 
+	/// Returns the number of components of the process variable.
+	int getNComponents() const { return _n_components; }
```

Comment:
In the property vector the method is called `getNumberOfComponents()`. Maybe you want to use the longer name here, too.